addons/sourcemod/scripting/ZR_RepeatKillDetector.sp

```diff
@@ -18,12 +19,31 @@ float g_fRepeatKillDetectThreshold;
 
 public Plugin myinfo = {
 	name = "[ZR] Repeat Kill Detector",
-	author = "GoD-Tony + BotoX + .Rushaway",
+	author = "GoD-Tony + BotoX + .Rushaway, Vauff",
 	description = "Disables respawning on maps with repeat killers",
-	version = "1.1.0",
+	version = ZR_RKD_VERSION,
 	url = "http://www.sourcemod.net/"
 };
 
+public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max)
+{
+	CreateNative("RepeatKillerEnabled", Native_RepeatKillerEnabled);
+	RegPluginLibrary("ZR_RepeatKillDetector");
+	return APLRes_Success;
+}
+
+public void OnPluginStart()
+{
+	RegAdminCmd("zr_killrepeator", Command_ForceRepeator, ADMFLAG_BAN, "Enable or Disable respawning for this round.");
+
+	RegAdminCmd("sm_togglerepeatkill", Command_RkON, ADMFLAG_GENERIC, "Turns off the repeat killer detector if it is enabled");
```

Comment:
why so many commands for the same thing?

just keep one command that enable/disabled it

Reply:
Admins complain about the command complexity to write. Just QoL improvements.

Reply:
Because typing `!rkon` and `!rkoff` is faster than typing `!killrepeator 1/0`, which is a QoL change I wanted to see.

Reply:
i see so maybe just keep !rk instead and if no arg its toggle either on / off or !rk 0/1

Reply:
The goal behind all these commands is to let donated able to turn off, but not to turn on.

Reply:
oh okay fair point

Reply:
so maybe only keep !rk* commands and remove sm_toggle*